UI/window-basic-settings.cpp

```diff
@@ -715,6 +715,24 @@ OBSBasicSettings::OBSBasicSettings(QWidget *parent)
 	UpdateAutomaticReplayBufferCheckboxes();
 
 	App()->EnableInFocusHotkeys(false);
+	// bolden headers of Settings > General  & Settings > Advanced
+	ui->groupBox_10->setStyleSheet("QGroupBox { font-weight: bold; } ");
```

Comment:
Can this be done in the layout file?

Reply:
It is possible to place it in the stylesheet file (the place where it should be). Here, only style should be updated (if element created dynamically and it has its own special settings).